blocks/links-bar-block/features/links-bar/default.jsx

```diff
@@ -33,23 +32,34 @@ const LinksBar = ({ customFields: { navigationConfig = {}, ariaLabel } }) => {
 
 	return (
 		<>
-			<nav
+			<Stack
+				className={BLOCK_CLASS_NAME}
+				justification="center"
+				direction="row"
+				as="nav"
 				key={id}
-				className="links-bar"
 				aria-label={ariaLabel || phrases.t("links-bar-block.element-aria-label")}
 			>
 				{menuItems &&
 					menuItems.map((item, index) => (
-						<PrimaryFont as="span" className="links-menu" key={item._id}>
+						<React.Fragment key={item._id}>
 							{item.node_type === "link" ? (
-								<Link href={item.url} name={item.display_name} />
+								<Link href={item.url} className={`${BLOCK_CLASS_NAME}__menu-link`}>
```

Comment:
If there is only ever one link style in this block, you can style the link with the component child selector in the tokens:
```
    blocks: (
        links-bar: (
            components: (
                link: (
                    color: red,
                )
            )
        ),
    ),
```
Adding a class `__menu-link` is redundant.

Reply:
So you should not need to add this class.